packages/babel-plugin-transform-named-capturing-groups-regex/test/fixtures/wrapper/basic/output.js

```diff
@@ -1,3 +1,3 @@
-"foo".match(babelHelpers.wrapRegExp(/(.)\1/, {
+"foo".match(babelHelpers.wrapRegExp(/([\0-\t\x0B\f\x0E-\u2027\u202A-\uFFFF])\1/, {
```

Comment:
This is an known issue for regexpu, it would produce suboptimal output for dot: https://github.com/mathiasbynens/regexpu-core/issues/24

Reply:
Fixed at https://github.com/mathiasbynens/regexpu-core/pull/29